src/seedu/addressbook/AddressBook.java

```diff
@@ -1037,7 +1033,7 @@ private static boolean isPersonDataExtractableFrom(String personData) {
 	 *            person string representation
 	 * @return name argument
 	 */
-	private static String extractNameFromPersonString(String encoded) {
+	private static String extractName(String encoded) {
```

Comment:
The new method has a generic name yet it cannot extract the Name if the String format changes. Is this really needed?